doc/source/contributing_code.rst

```diff
@@ -202,39 +199,77 @@ This sets the number of files to 4096. However, this command only applies to you
 Building MDAnalysis
 -------------------
 
-Make sure that you have :ref:`cloned the repository <forking-code-repo>`  
-and activated your virtual environment. First we need to install dependencies. If you're using conda, you'll need a mix of conda and pip installations:
+With conda
+----------
+
+.. note::
+    Make sure that you have :ref:`cloned the repository <forking-code-repo>`  
+    and activated your virtual environment with :code:`conda activate mdanalysis-dev`.
+    
+First we need to install dependencies. You'll need a mix of conda and pip installations:
 
     .. code-block:: bash
 
         conda install -c biobuilds -c conda-forge \
             biopython chemfiles clustalw==2.1 codecov cython \
             griddataformats gsd hypothesis "joblib>=0.12" \
             matplotlib mmtf-python mock netcdf4 networkx \
-            "numpy>=1.17.3" psutil pytest scikit-learn scipy \
+            "numpy>=1.18.0" psutil pytest scikit-learn scipy \
             "seaborn>=0.7.0,<0.9" sphinx==1.8.5 "tidynamics>=1.0.0" \
-            "tqdm>=4.43.0"
-
-        # if using conda with python 3.7 or 3.8, also run
-        conda install -c conda-forge parmed
-
-        # if using conda with other versions of python, also run
-        pip install parmed
+            "tqdm>=4.43.0" parmed
 
         # documentation dependencies
         pip install sphinx-sitemap sphinx_rtd_theme msmb_theme==1.2.0
 
-If you're using pip, it is a little simpler. However, some packages such as ``clustalw`` are not available via pip.
+Ensure that you have a working C/C++ compiler (e.g. gcc or clang). You will also need Python ≥ 3.7. We will now install MDAnalysis. 
+
+    .. code-block:: bash
+
+        # go to the mdanalysis source directory
+        cd mdanalysis/
+
+        # Build and install the MDAnalysis package
+        cd package/
+        pip install -e .
+        
+        # Build and install the test suite
+        cd ../testsuite/
+        pip install -e .
+
+At this point you should be able to import MDAnalysis from your locally built version. If you are running the development version, this is visible from the version number ending in :code:`-dev0`. For example:
+
+    .. code-block:: bash
+
+        $ python  # start an interpreter
+        >>> import MDAnalysis as mda
+        >>> mda.__version__
+        '2.1.0-dev0'
```

Comment:
Would `python -c "import MDAnalysis as mda; print(mda.__version__)"` be more foolproof?

Reply:
Yep, I like it!